internal/pkg/terraform/validators.go

```diff
@@ -0,0 +1,12 @@
+package terraform
+
+import (
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+)
+
+var LowercaseRegexValidator = stringvalidator.RegexMatches(
+	regexp.MustCompile(`^(([^a-zA-Z0-9])|([a-z0-9]))*$`),
```

Comment:
Slugs should alphanumeric not non alphanumeric. I think this will allow things like `@!` . Ref https://www.npmjs.com/package/slugify